website/content/en/highlights/2022-03-22-0-21-0-upgrade-guide.md

```diff
@@ -28,6 +29,32 @@ We cover them below to help you upgrade quickly:
 
 ### Breaking changes
 
+#### Syntax changes for non-VRL paths {#path-syntax-changes}
+
+Previously, there were two different ways to describe paths. VRL (remap transforms) use a newer syntax, while everything
+else in Vector (templating / all transforms except remap) still used an older syntax. This was a constant pain point for users,
+and we have taken some steps to migrate towards the VRL syntax. This is a **breaking change** that may require migration.
```

Comment:
It might also be worth specifically calling out that Vector allows paths to start with or without a `.`, whereas VRL treats with/without dots differently (event query vs variable query).